new/rfcXXX-telemetry.md

```diff
@@ -0,0 +1,88 @@
+---
+RFC: unassigned
+Author: Ranjib Dey <ranjib@linux.com>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Chef telemetry
+
+## Motivation
+
+  As a chef user and developer
+  I want an easy way to get chef run performance data
+  so that I can identify areas that can be optimized
+
+As we automate more things with chef and control more system resources,
+chef run time gets longer, chef client process consume more memory and cpu.
```

Comment:
"chef run time increases, and the chef client process consumes more memory and cpu."